packages/stats-py/tests/test_tier3.py

```
@@ -0,0 +1,190 @@
+        df,
+        {
+            "latents": {"X1": ["X1.1", "X1.2", "X1.3", "X1.4"], "Y": ["Y.1", "Y.2", "Y.3", "Y.4"]},
+            "paths": ["X1 -> Y"],
+        },
+    )
+    loadings = _table(result, "loadings")
+    # 8 item, 2 di antaranya loading referensi (difiksasi 1, tanpa SE).
+    assert len(loadings["rows"]) == 8
+    structural = _table(result, "structural")
+    assert len(structural["rows"]) == 1
+    source, target, estimate, std, se, z, p = structural["rows"][0]
```

Ruff (0.15.20)

[warning] Unpacked variable `se` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)